kernel/fablib/actions/host/exec.go

```diff
@@ -23,24 +23,24 @@ import (
 	"github.com/sirupsen/logrus"
 )
 
-func Exec(h *model.Host, cmd string) model.Action {
+func Exec(h *model.Host, cmds ...string) model.Action {
 	return &exec{
-		h:   h,
-		cmd: cmd,
+		h:    h,
+		cmds: cmds,
 	}
 }
 
 func (exec *exec) Execute(m *model.Model) error {
 	sshConfigFactory := fablib.NewSshConfigFactoryImpl(m, exec.h.PublicIp)
 
-	if o, err := fablib.RemoteExec(sshConfigFactory, exec.cmd); err != nil {
+	if o, err := fablib.RemoteExecSeq(sshConfigFactory, exec.cmds...); err != nil {
```

Comment:
`Seq`?

Reply:
Sequential (as opposed to parallel)

Reply:
But that kind of works backwards of the `Sequential`/`Parallel` in the lifecycle stages. In those you say things like `rsync.Parallel`. Here, you're using 1 `RemoteExec` and passing it multiple things to run sequentially? Just feels kind of strange.

I kind of like `RemoteExecAll`, which implies that you're just batching multiple commands.